src/main/java/fruit/shop/strategy/PurchaseOption.java

```diff
@@ -0,0 +1,12 @@
+package fruit.shop.strategy;
+
+public class PurchaseOption implements Option {

```

Comment:
more idiomatic way would be to name it PurchaseHandler implements Handler, Option is a bit confusing